src/community/params-extractor/src/main/java/org/geoserver/params/extractor/RequestWrapper.java

```diff
@@ -91,15 +93,15 @@ public String getParameter(String name) {
     }
 
     private String extractPathInfo(String requestUri) {
-        Matcher matcher = pathInfoPattern.matcher(requestUri);
+        Matcher matcher = pathsPattern.matcher(requestUri);
```

Comment:
Basically these methods are now repeating the same thing, wouldn't it make more sense to build the matcher just one in the calling method, set the variables accordingly, and remove these two methods?

Reply:
Indeed, moved to root caller method, also maintaining pattern object as class field not needed anymore.